google/cloud/firestore_v1/collection.py

```diff
@@ -121,14 +119,22 @@ def list_documents(self, page_size=None) -> Generator[Any, Any, None]:
         )
         return (_item_to_document_ref(self, i) for i in iterator)
 
-    def get(self, transaction=None) -> Generator[document.DocumentSnapshot, Any, None]:
-        """Deprecated alias for :meth:`stream`."""
-        warnings.warn(
-            "'Collection.get' is deprecated:  please use 'Collection.stream' instead.",
-            DeprecationWarning,
-            stacklevel=2,
-        )
-        return self.stream(transaction=transaction)
+    def get(self, transaction=None) -> list:
+        """Read the documents in this collection.
```

Comment:
oh, wait, did we undeprecate this? memory is fuzzy.

Reply:
Yep, it looks like it was undeprecated after the `v2-staging` split.

Reply:
As seen on [master](https://github.com/googleapis/python-firestore/blob/master/google/cloud/firestore_v1/query.py#L784-L820)